### ISSUE TITLE
Load multiple scripts simultaneously

### ISSUE THREAD

**Issue by fairking** [tserkov/vue-plugin-load-script#15]
Is there any way to load several scripts simultaneously?
At the moment I can only do this:
```
this.$loadScript("/js/script1.js").then(() => {
    this.$loadScript("/js/script2.js").then(() => {
        this.$loadScript("/js/script3.js").then(() => {
            // Do something after all 3 scripts loaded.
        }).catch(() => {
            // Failed to fetch script3
        });
    }).catch(() => {
        // Failed to fetch script2
    });
}).catch(() => {
    // Failed to fetch script1
});
```
But this solution has one big downside. Befole loading script2 it will wait for the script1. It increases the loading time dramatically.

It would be nice to have something like this where all scripts are loaded asynchniously and only when all of them are loaded the `then` is called:
```
this.$loadScript(["/js/script1.js", "/js/script2.js", "/js/script3.js"]).then(() => {
    // Do something after all 3 scripts loaded.
}).catch(() => {
    // Failed to fetch at least one of those scripts
});
```

**Comment by tserkov** [comment]
That isn't currently possible, but it sounds like great new functionality to add!

**Comment by victornpb** [comment]
I don't think this is necessary since you could easily do this, and all of them would be downloaded in parallel by the browser. It will fail if one of them fail. 

```js
Promise.all([
    this.$loadScript(),
    this.$loadScript(),
    this.$loadScript()
]).then(()=>{
    // something after all 3
})
```

or even 

```js
await Promise.all([
    this.$loadScript(),
    this.$loadScript(),
    this.$loadScript()
]);

// something after all 3 
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

**Comment by tserkov** [comment]
Recommending using `Promise.all` for multiple scripts, as detailed in the [above comment](https://github.com/tserkov/vue-plugin-load-script/issues/15#issuecomment-647834889).
